script.skinvariables/resources/lib/script.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
you should really try to make this file as smaller as possible (the only file that is not compiled to bytecode as this is an entrypoint). As this is a skin helper script it is called all the type.

Can you refactor to make something like:

```python
import sys
from resources.lib import myscriptlogic

myscriptlogic.run(sys.argv)
```

Reply:
I'm confused... isn't this what the other script.py file does:
https://github.com/xbmc/repo-scripts/blob/b81e1700f8b982e34fb4a69e3aee44c5c28f260f/script.skinvariables/script.py

There's two files called `script.py` - one in the basedir and one in `resources/lib/`
The one in the basedir is super simple and is literally just:
```python
from resources.lib.script import Script

if __name__ == '__main__':
    TMDbScript = Script()
    TMDbScript.get_params()
    TMDbScript.router()
```

Reply:
Disresgard, I though this was the entrypoint. All good